project/plugins.sbt

```diff
@@ -0,0 +1,7 @@
+addSbtPlugin("org.scoverage"             % "sbt-scoverage"% "1.6.0")
+addSbtPlugin("com.dwijnand"              % "sbt-dynver"   % "4.0.0")
+addSbtPlugin("ch.epfl.scala"             % "sbt-bloop"    % "1.3.5")
+//addSbtPlugin("org.scalameta"             % "sbt-scalafmt" % "2.2.1")
```

Comment:
Be good to see scalafmt enabled before merge

Reply:
👍